ui_framework/components/table/table_row_cell_liner.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+export const KuiTableRowCellLiner = ({ children, className, ...rest }) => {
+  const classes = classNames('kuiTableRowCell__liner', className);
+  return <div className={classes} {...rest} >{children}</div>;
+};
+KuiTableRowCellLiner.propTypes = {
+  children: PropTypes.node,
+  className: PropTypes.string,
+};
+
+
+
```

Comment:
Does your IDE support a setting for ensuring that the file [ends with a single newline](https://github.com/airbnb/javascript#whitespace--newline-at-end)?